src/code/z_actor.c

```diff
@@ -1889,24 +1912,24 @@ s32 func_800B8500(Actor* actor, PlayState* play, f32 xzRange, f32 yRange, Player
 
     player->talkActor = actor;
     player->talkActorDistance = actor->xzDistToPlayer;
-    player->exchangeItemId = exchangeItemId;
+    player->exchangeItemAction = exchangeItemAction;
 
     CutsceneManager_Queue(CS_ID_GLOBAL_TALK);
     return true;
 }
 
-s32 func_800B85E0(Actor* actor, PlayState* play, f32 radius, PlayerItemAction exchangeItemId) {
-    return func_800B8500(actor, play, radius, radius, exchangeItemId);
+s32 Actor_OfferTalkExchangeRadius(Actor* actor, PlayState* play, f32 radius, PlayerItemAction exchangeItemAction) {
+    return Actor_OfferTalkExchange(actor, play, radius, radius, exchangeItemAction);
```

Comment:
Technically, the radius here is the radius defined on an "Equilateral Cylinder" (the blue shape below) since `xz` and `y` are measured separately. I think `radius` is the right term for that (you won't find a better term I don't think), but it might be worth a comment or an `@radius` that this is the radius of an equilateral cylinder?
https://en.wikipedia.org/wiki/Right_circular_cylinder#:~:text=.-,Equilateral%20cylinder,-%5Bedit%5D

![image](https://github.com/zeldaret/mm/assets/47598039/fd04fe35-ba44-4c2d-b711-ad1aff3881db)

Reply:
`Actor_OfferTalkExchangeRadius` sounds fine to me, we could explain the cylinder stuff in a comment

Reply:
what makes this radius special that it deserves to be in the name vs `Actor_OfferTalkExchange` which also has a radius? This is my main point

Reply:
Another way of phrasing, the radius isnt what makes it special, its the fact that the value gets used for both the radius and height of the cylinder (vs `Actor_OfferTalkExchange` which has separately configurable radius and height). Im hoping to find a name that better reflects this

Reply:
Tbh, I actually think `Actor_OfferTalkExchangeEquiCylinder` isn't too bad. A bit lengthy, but does seem better than `Radius` to get why this one is different to the original `Actor_OfferTalkExchange`.